### ISSUE TITLE
Search field missing for newly opened windows

### ISSUE THREAD

**Issue by julian-weinert** [yuhua-chen/MCLog#41]
When opening new windows using `⌘-⇧-T`, the search field doesn't get added automatically. Instead I need to open the new window, quit Xcode and reopen.

Enhancement would be highly appreciated.

**Comment by yuhua-chen** [comment]
Does it happen all the time? Somehow I can reproduce this issue but mostly I can't. for now, I can't listen any better notifications than current ones when a window opens.

**Comment by julian-weinert** [comment]
Yep. Whenever I restart Xcode after opening a new window, it is there, whenever Xcode is running while doing so I have the problem. Reproducibility :100:

**Comment by yuhua-chen** [comment]
Got it. I will try to find out if there's another way to listent opening window event.

**Comment by julian-weinert** [comment]
I already started digging in the code. But unfortunately I don't have enough time to contribute myself at the moment...

**Comment by yuhua-chen** [comment]
Try out the latest commit(5761063) to see if the search filed shows up on different windows.

**Comment by yuhua-chen** [comment]
How did it go? if there's no question, i'm about to close this issue.
